### PR TITLE
Add email, address and note to QR code

### LINKED ISSUE #146: Enhancement: Export more information using QR code
Currently, the QR code only exports the contact name and number.

Would be nice if it was able to export more attributes as well.

### PR DESCRIPTION
**Pull request by dillontkh**
Resolves #146

### CODE REVIEW

File: src/main/java/seedu/address/logic/QrCodeGenerator.java [hunk 1]
```diff
@@ -68,10 +68,13 @@ public static void generateQrCode(Person person) throws WriterException, IOExcep
...
     private static String createVCardString(Person person) {
         return "BEGIN:VCARD\n"
-                + "VERSION:3.0\n"
+                + "VERSION:4.0\n"
                 + "N:" + person.getName() + ";;;\n"
                 + "FN:" + person.getName() + "\n"
                 + "TEL;TYPE=CELL:" + person.getPhone() + "\n"
+                + "EMAIL:" + person.getEmail() + "\n"
+                + "ADR;TYPE=HOME:;;" + person.getAddress() + "\n"
+                + "NOTE:" + person.getNote() + "\n"
```
Comment (bgopi23) [inline src/main/java/seedu/address/logic/QrCodeGenerator.java:77]: very nice formatting!

**Review by bgopi23 — APPROVED** [review]
LGTM!

**Review by jjchee77 — APPROVED** [review]
LGTM! Might want to resolve #177 as part of this PR

**Comment by dillontkh** [comment]
> LGTM! Might want to resolve #177 as part of this PR

Fixed, thanks for the catch!

**Pull request merged**